shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -62,6 +62,14 @@ class MealApi {
         }
     }
 
+    suspend fun reserveAMeal(id: String): Meal {
+        var meal = getMeal(id)
```

Comment:
Does this need to be var? i can't see it being reassigned anywhere in the function